packages/odyssey-react-mui/src/index.ts

```diff
@@ -10,5 +10,80 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+export {
+  Alert,
```

Comment:
Correct! But it also changes how you setup `ThemeProvider`s. Because we're re-exporting MUI, you'll need a separate `ThemeProvider` (like you have today) for MUI imports versus Odyssey imports. The difference is that you _also_ need the Odyssey `ThemeProvider`.